Samples/GStreamer/Play.cs

```diff
@@ -7,11 +7,11 @@ public class Gst
         public static void Play()
         {
             Application.Init();
-            var ret = Parse.Launch("playbin uri=playbin uri=http://ftp.halifax.rwth-aachen.de/blender/demo/movies/ToS/tears_of_steel_720p.mov");
+            Element ret = Parse.Launch("playbin uri=playbin uri=http://ftp.halifax.rwth-aachen.de/blender/demo/movies/ToS/tears_of_steel_720p.mov");
```

Comment:
See above comment. We need to ensure that `Element` is not null before using it.

Reply:
Not needed as `Parse.Launch` now throws an exception which would lead to a program crash if thrown. But in case of an exception the users receives a detailed message what went wrong.

Reply:
This is fine for me, but w.r.t what I described in #93 about GError. If I intentionally break this (e.g. rename `playbin` to `plybin`), I get the following error message:

```
PS D:\gir.core\Samples\GStreamer> dotnet run
Starting to play tears of steal. Please wait while file is beeing loaded...
Unhandled exception. GLib.GException: ?????????"?
   at Gst.Parse.Launch(String pipelineDescription) in D:\gir.core\Libs\Gst\Classes\Parse.cs:line 22
   at Sample.Gst.Play() in D:\gir.core\Samples\GStreamer\Play.cs:line 13
   at Samples.Program.Main(String[] args) in D:\gir.core\Samples\GStreamer\Program.cs:line 7
```

The exception message is garbled because of the way we're marshalling it. This problem is unrelated to this PR, but means we definitely need to look into `Error` and `PtrToStringAnsi` vs `PtrToStringAuto`.